classes/patterns/class-realitycheck.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Reality Check pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+/**
+ * Class Reality Check.
+ *
+ * @package P4GBKS\Patterns
+ */
+class RealityCheck extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/reality-check';
+	}
+
+	/**
+	 * Returns the template for one column.
+	 */
+	public static function get_column_template(): string {
+		return '
+			<!-- wp:column -->
+				<div class="wp-block-column">
+					<!-- wp:image {"align":"center","className":"mb-0 force-no-lightbox"} -->
+						<div class="wp-block-image mb-0 force-no-lightbox">
+							<figure class="aligncenter">
+								<img alt="" />
+							</figure>
+						</div>
+					<!-- /wp:image -->
+					<!-- wp:heading {"style":{"typography":{"fontSize":"64px"}},"align":"center","className":"has-text-align-center mb-0","placeholder":"' . __( 'Enter title', 'planet4-blocks-backend' ) . '"} -->
+						<h2 style="font-size:64px;" class="has-text-align-center mb-0"></h2>
```

Comment:
I'm always confused, should we express font-size in `rem` or `px` ?

Reply:
We use a mix of both in our codebase, so it is confusing 😬 But I think ideally we should use [our fonts](https://github.com/greenpeace/planet4-master-theme/blob/master/assets/src/scss/base/_fonts.scss) (and therefore `rem`) when possible... Here however we are using WordPress' native functionality and that uses pixels, so we don't really have a choice:

<img width="293" alt="Screenshot 2022-03-31 at 10 25 44" src="https://user-images.githubusercontent.com/6949075/161011288-76ad0e00-e1b6-44cf-924d-4d8c5dd12efc.png">

Reply:
Hooo didn't realize that, good to know thanks :+1:

Reply:
Nevermind what I said, turns out you can select a unit there so I will ask the design team what they would rather use! Good catch @lithrel 👍

Reply:
@lithrel I've updated the pattern to use `rem` after discussing with Mag